src/routes/forms.js

```diff
@@ -181,18 +164,9 @@ export default [
     async handler(request) {
       const { params } = request
       const { id } = params
+      const definition = await getFormDefinition(id, 'live')
```

Comment:
Can ditch async + await here too 💪

```suggestion
      return getFormDefinition(id, 'live')
```

Reply:
Done